### DISCUSSION TITLE
.

### DISCUSSION THREAD

**Discussion by ghost** [quoid/userscripts#391] [General]
.

**Comment by ACTCD** [comment]
The project authors have given [clear advice](https://github.com/reek/anti-adblock-killer/wiki). If you really took the time to research, you must have been read this page.

I don't know why you're still stuck with this script that hasn't been updated in over 6 years. Ad blocking is a job that needs to be kept up-to-date, and from any perspective you should not expect a script that is no longer updated.

**Comment by quoid** [comment]
macOS Safari and iOS Safari browsers are **not the same**. Just because something works on macOS, it does not mean it is guaranteed to work on iOS. You've asked multiple times why this is happening - *we do not know* and do not have the resources to debug this for you. 

This is an issue to address with [script author](https://github.com/reek/anti-adblock-killer) and if he is not responding, it is not our responsibility to resolve issues with his project.
